src/RootedTrees.jl

```diff
@@ -1413,6 +1415,42 @@ function butcher_representation(t::RootedTree, normalize::Bool = true)
     return result
 end
 
+"""
+    elementary_differential(t::RootedTree)
+
+Returns the elementary differential as a LaTeXString.
+
+"""
+function elementary_differential(t::RootedTree)
+    return latexstring(rec_elementary_differential(t))
+end
+
+"""
+Function used to go recursively through the RootedTree to generate the elementary differential of the tree.
+"""
+function rec_elementary_differential(t::RootedTree)
+    subtree_strings = []
+    for subtree in SubtreeIterator(t)
+        push!(subtree_strings,rec_elementary_differential(subtree))
```

Comment:
Please run the JuliaFormatter locally on your PR. If you have installed [JuliaFormatter.jl](https://github.com/domluna/JuliaFormatter.jl) in your default environment, this can be done by

```julia
julia> using JuliaFormatter; format(".")
```

if you are in the directory of this package.